src/Pages/Categories.jsx

```diff
@@ -0,0 +1,19 @@
+import { Link} from 'react-router-dom';
+import { Flex, Box } from '@chakra-ui/react'
+
+export const Categories = () => {
+    return (
+      <>
+      <Box bg='orange.300' w='100%' p={4} color='cornsilk'>
+      <Flex direction='column' justify='center' align='center'>
+      <main>
+          <h2>Welcome to the categories!</h2>
+        </main>
+        <nav>
+          <Link to='/'>Home</Link>
+        </nav>
+        </Flex>
+      </Box>
+      </>
+    );
+}
```

Comment:
For some reason pre-commit hook with code formatting doesn't work on your machine.
Please run `npm run format` and commit the result.

Also try to update all dependencies by removing "node_modules" folder and running `npm i` again.

Reply:
forgot, sorry :) fixed